tests/integration/integration_test_case.py

```diff
@@ -167,7 +167,8 @@ def get(self, url, follow_redirects=True, **kwargs):
         :param url: the URL to GET
         """
         response = self._client.get(url, follow_redirects=follow_redirects, **kwargs)
-
+        if last_cookie is not None:
+            response.headers["Set-Cookie"] = last_cookie
```

Comment:
Can we not store the last cookie on the instance?. That way we don't have to pass it into the "get" method.

Reply:
Its been a while since I made this change so I will have to have a look 👍

Reply:
I had a look, I don't think I can cause the header doesn't exist for the `Set-Cookie` unless you save it prior to the `getUrlCookie`. I am not entirely sure if thats what you meant , sorry 😅

Reply:
I mean store it on the instance like we do with `last_csrf_token` but when you do get a cookie, update it.

```py
class IntegrationTestCase(unittest.TestCase):  # pylint: disable=too-many-public-methods
    def setUp(self):
        # Cache for requests
        self.last_url = None
        self.last_response = None
        self.last_csrf_token = None
        self.redirect_url = None
        self.last_response_headers = None
```

So another attribute here. Then in `get(...)`, if the response header has `Set-Cookie` set `self.cookie` to the header cookie value.

Reply:
Updated and comment added 👍